superset/assets/javascripts/explorev2/index.jsx

```diff
@@ -32,7 +32,8 @@ const bootstrappedState = Object.assign(
     user_id: bootstrapData.user_id,
     chartUpdateStartTime: now(),
     chartUpdateEndTime: null,
-    chartStatus: 'loading',
+    chartStatus: 'trigger',
```

Comment:
oops, that was leftover from a previous approach before I decided to use an action creator to run queries, removing...